pkg/piperutils/slices.go

```diff
@@ -94,7 +95,9 @@ func SplitAndTrim(in []string, separator string) (out []string) {
 	return
 }
 
-// UniqueStrings removes duplicates from values
+// UniqueStrings removes duplicates from values. The order of the values
```

Comment:
I wouldn't expect, that the order is changed:
b,a,c -> b,a,c
b,b,a,c -> b,a,c
b,a,c,a -> b,a,c | b,c,a  
sorting could be done by the consumer, solely in case of duplicates the method needs to take a decision to remove the first, or last entry.

Reply:
I think without sorting we cannot make any assumtion wrt order. Internally we create a Set and add all the entries to the set. Afterwards we create another slice and add all the entries of the set. Again: to my understading we cannot make any assuption wrt order.

I have no problem with simply closing this PR. In that case we can add some statement to the docu explaining that there is no assumption wrt order.

Having some additional coding for ensuring the same order like before, but without duplicates looks like overkill for me.